haskell-src/lib/ChainwebData/Env.hs

```diff
@@ -224,12 +234,47 @@ data FillArgs = FillArgs
   { _fillArgs_delayMicros :: Maybe Int
   } deriving (Eq, Ord, Show)
 
-data ServerEnv = ServerEnv
-  { _serverEnv_port :: Int
-  , _serverEnv_runFill :: Bool
-  , _serverEnv_fillDelay :: Maybe Int
-  , _serverEnv_serveSwaggerUi :: Bool
-  } deriving (Eq,Ord,Show)
+data WorkerEnv = Full FullEnv | HTTP HTTPEnv | ETL ETLEnv
+  deriving (Eq,Ord,Show)
+
+type FullEnv = (HTTPEnv, ETLEnv)
+
+_Full :: Prism' WorkerEnv FullEnv
+_Full = prism' Full $ \case
+  Full x -> Just x
+  _ -> Nothing
+
+_HTTP :: Prism' WorkerEnv HTTPEnv
+_HTTP = prism' HTTP $ \case
+  HTTP x -> Just x
+  _ -> Nothing
+
+_ETL :: Prism' WorkerEnv ETLEnv
+_ETL = prism' ETL $ \case
+  ETL x -> Just x
+  _ -> Nothing
+
+data HTTPEnv = HTTPEnv
+ { _httpEnv_port :: Int
+ , _httpEnv_serveSwaggerUi :: Bool
+ } deriving (Eq,Ord,Show)
+
+httpEnv_port :: Lens' HTTPEnv Int
+httpEnv_port = lens _httpEnv_port $ \x y -> x { _httpEnv_port = y }
+
+httpEnv_serveSwaggerUi :: Lens' HTTPEnv Bool
+httpEnv_serveSwaggerUi = lens _httpEnv_serveSwaggerUi $ \x y -> x { _httpEnv_serveSwaggerUi = y }
```

Comment:
Do we really need these lenses? By nature, these `XXXEnv` values are meant to be read by the rest of the codebase and not really manipulated, so I think we could just use the field accessors instead of lenses.